README.md

```diff
@@ -661,6 +661,7 @@ array expressions, inspired by NumPy syntax. [BSD 3-clause] [website](http://qua
 * [CPFG](https://github.com/cpgf/cpgf) - A C++03 library for reflection, callback and script binding. [Apache2] [website](http://www.cpgf.org/)
 * [CPP-Reflection](https://github.com/AustinBrunkhorst/CPP-Reflection) - C++ Reflection using clang. [MIT]
 * [Nameof](https://github.com/Neargye/nameof) - Header-only C++17 library provides nameof macros and functions to obtain the simple name of variable, type, function, macro, and enum. [MIT]
+* [Magic Enum](https://github.com/Neargye/magic_enum) Header-only C++17 library provides static reflection for enums (to string, from string, iteration), work with any enum type without any macro or boilerplate code. [MIT]
```

Comment:
Hi @Neargye Thanks for the addition. How about moving this line up so that these items are ordered alphabetically?

Reply:
Thanks, fixed this.

Reply:
@Neargye Thanks! :-)